docs/src/pages/guides/interoperability/interoperability.md

```diff
@@ -42,15 +43,15 @@ export default function PlainCssSlider() {
   return (
     <div>
       <Slider defaultValue={30} />
-      <Slider className="slider" defaultValue={30} />
+      <Slider defaultValue={30} className="slider" />
     </div>
   );
 }
 ```
 
-### Controlling priority ⚠️
+### CSS injection order ⚠️
 
-**Note:** Most CSS-in-JS solutions inject their styles at the bottom of the HTML `<head>`. If you don't want to mark style attributes with **!important**, you need to change the CSS injection order. Here's a demo of how it can be done for the default style engine - emotion:
+**Note:** Most CSS-in-JS solutions inject their styles at the bottom of the HTML `<head>`, which make Material-UI take precedence over your `<style>`. To remove the need for **!important**, you need to change the CSS injection order. Here's a demo of how it can be done for the default style engine - emotion:
```

Comment:
```suggestion
**Note:** Most CSS-in-JS solutions inject their styles at the bottom of the HTML `<head>`, which gives Material-UI precedence over your custom styles. To remove the need for **!important**, you need to change the CSS injection order. Here's a demo of how it can be done for the default style engine - emotion:
```